Apps/WebClient/src/ClientApp/src/views/ServicesView.vue

```diff
@@ -0,0 +1,46 @@
+﻿<script lang="ts">
+import Vue from "vue";
+import { Component } from "vue-property-decorator";
+
+import LoadingComponent from "@/components/LoadingComponent.vue";
+import BreadcrumbComponent from "@/components/navmenu/BreadcrumbComponent.vue";
+import BreadcrumbItem from "@/models/breadcrumbItem";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const options: any = {
+    components: {
+        BreadcrumbComponent,
+        LoadingComponent,
+    },
+};
+@Component(options)
+export default class ServicesView extends Vue {
+    private isLoading = true;
+    private breadcrumbItems: BreadcrumbItem[] = [
+        {
+            text: "Services",
+            to: "/services",
+            active: true,
+            dataTestId: "breadcrumb-services",
+        },
+    ];
+
+    private mounted(): void {
+        this.isLoading = false;
+    }
+}
+</script>
+
+<template>
+    <div>
+        <BreadcrumbComponent :items="breadcrumbItems" />
+        <LoadingComponent :is-loading="isLoading" />
+        <page-title title="Services"></page-title>
+        <h5 class="my-3">
+            You can check and update your Organ Donor Registry information here.
+            More health services will be added in future.
+        </h5>
```

Comment:
I'd suggest making this a `<p>`, but that's up to UX and the product owner.

Reply:
I followed your implementation from the dependents view:

https://github.com/bcgov/healthgateway/blob/dev/Apps/WebClient/src/ClientApp/src/views/DependentsView.vue#L162

Would you like me to change them both?

Reply:
Hi, @MikeLyttle,

I followed up with @mk08sh, regarding the styling etc. he's not worried about the current selection of styling as the text will be removed when more than one service is loaded. So I'm going to leave as is.